Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -11,354 +11,356 @@
 
 namespace Stratis.Bitcoin.Configuration
 {
-	public class NodeSettings
```

Comment:
Yes, tabs to spaces on NodeSettings and Node. Intentional on this class, but an accident on Node.cs

I also made some code-style fixes in NodeSettings.cs. If it helps make the changes clearer, I can recommit this classes without the spacing change.